package-res/resources/web/common-ui-require-js-cfg.js

```diff
@@ -63,6 +64,14 @@
   requirePaths["pentaho/_globalContextVars"] = basePath + "/pentaho/_globalContextVars";
   requirePaths["pentaho/contextVars"] = requirePaths["pentaho/_globalContextVars"];
 
+  // Singleton interface, virtual module ids
+  //requireMap["*"]["pentaho/spec/IContext"] = "pentaho/service!pentaho.spec.IContext?single";
+  requireMap["*"]["pentaho/config/IService"] = "pentaho/service!pentaho.config.IService?single";
+
+  // Default implementations
+  requireService["pentaho/config/impl/serviceInstance"] = "pentaho.config.IService";
+  requireService["pentaho/config/impl/AmdLoadedService"] = "Class<pentaho.config.IService>";
+
```

Comment:
Of interest here, and in moved files below, is the experiment of moving auxiliary classes that implement interfaces to a sub-package named `impl`. What do you think?